src/ServiceControl.Transports.ASBS/AzureQuery.cs

```diff
@@ -286,6 +291,37 @@ public override async IAsyncEnumerable<IBrokerQueue> GetQueueNames(
         throw new Exception($"Could not find a ServiceBus named \"{serviceBusName}\"");
     }
 
+    async Task<HashSet<string>> GetValidNamespaceNames(CancellationToken cancellationToken = default)
+    {
+        var validNamespaces = new HashSet<string>(StringComparer.OrdinalIgnoreCase) { serviceBusName };
+
+        // ArmEnvironment Audience Values: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager/src/ArmEnvironment.cs
+        // Service Bus Domains: https://learn.microsoft.com/en-us/rest/api/servicebus/
+        var serviceBusCloudDomain = armEnvironment.Audience switch
```

Comment:
Instead of a switch, we could use something like a dictionary to use the `ArmEnvironment` as the lookup key. That would avoid needing to hard code the management URLs as well as the domain sufffixes.